compiler/rustc_middle/src/lint.rs

```diff
@@ -227,6 +227,7 @@ pub fn explain_lint_level_source(
                     "`{} {}` implied by `{} {}`",
                     flag, hyphen_case_lint_name, flag, hyphen_case_flag_val
                 ));
+                err.help_once(format!("add an `#[allow({name})]` attribute to allow"));
```

Comment:
Could you not repeat the "allow" verb 2 times? It difficults readability. Maybe something like "to disable this warning"? Maybe "to accept this warning"?

Reply:
*to allow* is commonly used in the code base, so what about "use `#![allow({name})]` to allow" ?